src/js/utils/createCommandDirectionIcon.tsx

```diff
@@ -1,22 +1,41 @@
 import {ArrowDownward as ArrowDownwardIcon, QuestionMark as QuestionMarkIcon} from '@mui/icons-material';
 import {yellow} from '@mui/material/colors';
-import {analog} from '@jooby-dev/jooby-codec';
+import * as joobyCodec from '@jooby-dev/jooby-codec';
+import AnalogUnknownCommand from '@jooby-dev/jooby-codec/analog/UnknownCommand.js';
+import ObisObserverUnknownCommand from '@jooby-dev/jooby-codec/obis-observer/UnknownCommand.js';
 
+import {TCommandType} from '../types.js';
 
-const {directions} = analog.constants;
+import {COMMAND_TYPE_ANALOG, COMMAND_TYPE_OBIS_OBSERVER} from '../constants.js';
 
 
-const createCommandDirectionIcon = (directionType: typeof directions.DOWNLINK | typeof directions.UPLINK | undefined) => {
-    switch (directionType) {
-        case undefined:
-            return <QuestionMarkIcon sx={{mr: 2, color: 'grey.700'}}/>;
+const {directions} = joobyCodec.analog.constants;
 
-        case directions.UPLINK:
-            return <ArrowDownwardIcon sx={{mr: 2, transform: 'rotate(180deg)', color: `${yellow[800]}`}}/>;
+const unknownCommandClassMap = {
+    [COMMAND_TYPE_ANALOG]: AnalogUnknownCommand,
+    [COMMAND_TYPE_OBIS_OBSERVER]: ObisObserverUnknownCommand
+};
+
+const isUnknownCommandClassOrInstanceOf = (command: object, commandType: TCommandType) => {
+    const UnknownCommand = unknownCommandClassMap[commandType];
 
-        case directions.DOWNLINK:
-            return <ArrowDownwardIcon color="success" sx={{mr: 2}}/>;
+    if (command === UnknownCommand || command instanceof UnknownCommand) {
+        return true;
     }
+
+    return false;
+};
+
+const createCommandDirectionIcon = (command: object, commandType: TCommandType) => {
+    if (isUnknownCommandClassOrInstanceOf(command, commandType)) {
```

Comment:
is it really possible to have both UnknownCommand or it's instance here?

Reply:
discussed

also renamed `isUnknownCommandClassOrInstanceOf` -> `isUnknownCommandKind`